Client/DeviceCheck/DeviceCheck.swift

```diff
@@ -420,6 +392,10 @@ private extension DeviceCheckClient {
     case .setAttestation(_, let parameters):
       request.httpBody = try JSONEncoder().encode(parameters)
       request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+      
+    case .solveCaptcha(_, _, let nonce):
+      request.httpBody = try JSONSerialization.data(withJSONObject: ["solution": nonce])
```

Comment:
this nonce does not need percent encoding? i guess it's fine in http

Reply:
AFAIK it does not need to be percent encoded as its just a UUID string